README.md

```diff
@@ -19,3 +19,7 @@ quantitatively reason with them.
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+I believe that the Median-of-Three is a better choice for getting a better pivot. I think this because based on statistics as the bell curve, when choosing the lefmost as the pivot there is a 68% chance that it is in the middle quartile of the random permutation. To keep it simple we can say that it has about 1/2 chance of being a good pivot. For the median of three I got that there are 4 different versions that you could get being LLL = 1/64 LMR = 1/32 LMM = 1/16 and LRR = 1/64. This comes out to being a probability of 68.75% which does come out to be around 2/3
```

Comment:
Not sure if I follow you reasoning -- the probabilities you give don't add up to 0.6875, and you seem to be mixing cases that result in good and bad pivots.

Reply:
Oh ya your right, so are those three cases for bad pivots?

Reply:
This is your analysis, isn't it? I don't know what cases you've meant to consider.

Reply:
Well i am supposed to consider if it picks a piviot on the left side, middle, or right side and the middle is also split into two so left side would be 1/4 middle would be 2/4 or 1/2 and right side would be 1/4.  Ali said it had to start with a left side which only gives four options for what it could be, but are you saying that it could be 3 of any position, which would give me 10 options adding up to 60.38%?

Reply:
Why is the middle split into two? You need to analyze the possibilities where elements may come from and what the resulting pivot would be.

Reply:
In lab we split the middle in two because if there are three pivot spots one in the front one in the middle and one at the end there would be 4 subdivisions and so we made the middle two the middle making it 2/4 or 1/2.  I did calculate for each one though.  LLL = 1/64 LLM = 1/32 LLR = 1/64 RRR =1/64 RRM = 1/32 RRL = 1/64 MMM = 1/8 MML = 1/16 MMR = 1/16 and LMR = 1/32

Reply:
Again, I don't understand where you're going with this. LLL results in a bad pivot, MMM in a good pivot -- you can't consider them as the same.

Reply:
I see know, me and ali went through it again in lab and we took the four possibilities of good pivots which are LMR LMM MMR and MMM.  By calculating all the possible orderings of each of those pivots we get 12/64 + 12/64 +12/64 +8/64 = 44/64.  This gives us 68.75% which is around 2/3